src/hotspot/share/gc/shared/cardTableRS.hpp

```diff
@@ -92,18 +52,9 @@ class CardTableRS: public CardTable {
 
   virtual void verify_used_region_at_save_marks(Space* sp) const NOT_DEBUG_RETURN;
 
-  // Override.
-  void prepare_for_younger_refs_iterate(bool parallel);
-
-  // Card table entries are cleared before application;
-  void at_younger_refs_iterate();
-
   void inline_write_ref_field_gc(void* field, oop new_val) {
```

Comment:
It seems that the arg `new_val` is not used. Maybe remove it or add a comment saying it's an intentional omission.